Resources/Lang/en_US.json

```diff
@@ -526,7 +527,7 @@
   "PacifistInfo": "Vent to reset kill cooldowns",
   "RebirthInfo": "Arise Again",
   "MonarchInfo": "Give your crew extra voting power!",
-  "AbyssbringerInfo": "Create Black Holes",
+  "AbyssbringerInfo": "Place Black Holes",
```

Comment:
too long, did you check in game yourself?

Reply:
? im confusfed why you think it's too long bc there is more that are longer than that lol